packages/twenty-front/src/modules/ui/input/components/__stories__/Checkbox.stories.tsx

```diff
@@ -62,9 +66,14 @@ export const Catalog: CatalogStory<Story, typeof Checkbox> = {
           props: (shape: CheckboxShape) => ({ shape }),
         },
         {
-          name: 'variant',
-          values: Object.values(CheckboxVariant),
-          props: (variant: CheckboxVariant) => ({ variant }),
+          name: 'state',
```

Comment:
I would call the catalog dimension isHoverable ; state does not makes a lot of sense, it would be used to represent 'clicked' or 'selected' states

Reply:
updated, thanks!